templates/orgs_ext/org_home.haml

```diff
@@ -141,10 +153,27 @@
         .search-toolbar.clearfix
           - if perms.profiles.profile_user_create or org_perms.profiles.profile_user_create
             .pull-away
-              <a class="btn btn-default" href="{% url 'profiles.user_create' %}">{% trans "New" %}</a>
-
-          .{ style:"margin-top: 7px" }
-            Total of <strong>[[ users.length ]]</strong> users
+              .btn-group
+                %a.btn.btn-default{ href:"{% url 'profiles.user_create' %}" }
+                  - trans "New"
+                %button.btn.btn-default{ ng-click:"onExportUserStats()" }
+                  - trans "Export Users..."
+
+          .selection-controls.pull-back
+            .btn-group{ uib-dropdown:"true" }
```

Comment:
As I commented above, this would be better as a checkbox like "Include partner users" so that users can see what they are looking at without clicking a dropdown.

I think it should also default to false because some of our orgs are quite large, and fetching activity data for all users is an expensive query.